packages/web-component/src/lib/helpers/helpers.ts

```diff
@@ -330,10 +375,21 @@ export const getChromiumVersion = (
 // As an optimization - We can show first screen if we have startScreenId and we don't have oidcIdpStateId
 // - If there startScreenId it means that the sdk can show the first screen and we don't need to wait for the sdk to return the first screen
 // - If there is a oidcIdpStateId - we can't skip this call because the sdk may
+// - If there is a samlIdpStateId - we can't skip this call because the sdk may
+// - If there is a samlIdpUsername - we can't skip this call because the sdk may
+// - If there is a ssoAppId - we can't skip this call because the sdk may
```

Comment:
It's good that you added these comments, 
let's just make sure to complete the sentence :)

Reply:
Adjusted and Fixed :)